integration/test/validation/main_test.go

```diff
@@ -0,0 +1,86 @@
+// +build k8srequired
+
+package validation
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/giantswarm/apptest"
+	"github.com/giantswarm/micrologger"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/app-admission-controller/integration/env"
+)
+
+const (
+	prodCatalogName = "control-plane"
+	prodCatalogUrl  = "https://giantswarm.github.io/control-plane-catalog"
+	testCatalogName = "control-plane-test"
+	testCatalogUrl  = "https://giantswarm.github.io/control-plane-test-catalog"
```

Comment:
I was thinking if it would make sense to export these values as constants in the `apptest` library, instead of repeating them in every other test.

Reply:
Good idea, a list of known catalogs is nice. So we only need to specify the URL if its a community catalog.

I'll add that to apptest.

Reply:
Done in https://github.com/giantswarm/apptest/pull/17